test/images/README.md

```diff
@@ -0,0 +1,20 @@
+This image is the base image for Kubeflow Pipelines [Prow job](https://github.com/kubernetes/test-infra/blob/6555278147dfff550706b41c3f69f41ecf5a8c5a/config/jobs/kubeflow/kubeflow-postsubmits.yaml#L245). Including presubmit and postsubmit jobs.
+
+## How to build the image
+To build this Docker image, run the following Docker command from the directory containing the image's files:
+
+``` 
+docker build -t gcr.io/ml-pipeline-test/test-worker:latest . 
+```
+## Where to push the image
+This image is stored in the repo ml-pipeline-test, which is a public GCR repo. To push the image, run the following command:
```

Comment:
>repo ml-pipeline-test, which is a public GCR repo. 

Are you sure this is the case?

Reply:
Just changed that to public (~ 1 week ago). Do you have a better solution? Thanks!